### PR TITLE
[DRAFT][onert-micro] Revisit standalone CMakeLists.txt

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr makes changes to replace subdirectory in standalone CMakeLists.txt.

issue: https://github.com/Samsung/ONE/issues/9468

ONE-DCO-1.0-Signed-off-by: Artem Balyshev a.balyshev@partner.samsung.com

### CODE REVIEW

File: infra/onert-micro/CMakeLists.txt [hunk 1]
```diff
@@ -98,4 +98,6 @@ if(ENABLE_COVERAGE)
     target_link_libraries(onert_micro_coverage INTERFACE gcov)
 endif(ENABLE_COVERAGE)
 
-add_subdirectory("${NNAS_PROJECT_SOURCE_DIR}/onert-micro" "${CMAKE_BINARY_DIR}/onert-micro")
+if(NOT DISABLE_ONERT_MICRO_DIR)
```
Comment (binarman) [inline infra/onert-micro/CMakeLists.txt:101]: This part is needed to break `add_subdirectory` loop:
`infra/onert-micro/CMakeLists.txt` -> `onert-micro/CMakeLists.txt` -> `onert-micro/standalone/CMakeLists.txt` -> `infra/onert-micro/CMakeLists.txt`

I think, it is better to move some code from this file into separate files in infra.

For example `infra/onert-micro/cmake/utils.cmake` and/or `infra/onert-micro/cmake/variables.cmake` and include these files from `infra/onert-micro/CMakeLists.txt` and `infra/onert-micro/standalone/CMakeLists.txt`

**Comment by binarman** [comment]
@hseok-oh 
Hi!
Should we add you to `infra/onert-micro` related PRs?

**Comment by seanshpark** [comment]
I hope to split `infra` and `onert-micro` changes to separate PR.

**Comment by hseok-oh** [comment]
@binarman 

>Should we add you to `infra/onert-micro` related PRs?

No.  Current `/infra/onert-micro` is independent with other project's build.

File: infra/onert-micro/CMakeLists.txt (hunk 1 above)
Comment (BalyshevArtem) [reply infra/onert-micro/CMakeLists.txt]: Done

**Comment by BalyshevArtem** [comment]
> I hope to split `infra` and `onert-micro` changes to separate PR.

@binarman, I converted this PR as a draft, and if everything OK with it, I will split it into two: for `infra` and for `onert-micro`

**Comment by binarman** [comment]
> @binarman, I converted this PR as a draft, and if everything OK with it, I will split it into two: for infra and for onert-micro

Looks good to me )

**Comment by BalyshevArtem** [comment]
everything merged, close is

**Pull request closed without merging**